README.md

```diff
@@ -132,6 +133,7 @@ For this reason the following instrumentations are disabled by default:
 - `jdbc-datasource` which creates spans whenever `java.sql.DataSource#getConnection` method is called.
 - `servlet-filter` which creates spans around Servlet Filter methods.
 - `servlet-service` which creates spans around Servlet methods.
+- `jersey` which creates server spans in front of JAX-RS resources.
```

Comment:
what's the downside to enabling this by default? it looks like it will no-op if it's not needed (e.g. being used inside of a servlet container)?

Reply:
But it still will try to extract remote context from the request. Extra CPU cycles, more overhead.

Reply:
Ok, makes sense.

Why would someone enable this instrumentation over the grizzly instrumentation to solve #473?

Reply:
Honestly, I don't know :) This request came to me from my employer (Splunk) side. But potentially one can deploy JAX-RS in some other runtime, which we don't instrument yet.

Reply:
Can you ask for more details? It would be nice to have a concrete reason why we need it :smile:.

Reply:
I will, but do you have any reservations about having this merged? :)

Reply:
This was an upstream request from Splunk's own agent with the following rationale:

>This landed before we adopted upstream’s grizzly instrumentation, which I understand is often used w/ jersey apps though a lot of other servers are also compatible.  Unless the OTel community specifically doesn’t want this for some reason I don’t see a disadvantage of porting

Reply:
Any new instrumentation that we add has ongoing maintenance and support cost to this project, so I'm pretty confused why we would add new instrumentation if no one has a use case for it?

Reply:
I will talk to the people of my side.